include/rocksdb/snapshot.h

```diff
@@ -19,9 +19,13 @@ class DB;
 // To Destroy a Snapshot, call DB::ReleaseSnapshot(snapshot).
 class Snapshot {
  public:
-  // returns Snapshot's sequence number
+  // Returns Snapshot's sequence number.
```

Comment:
This line of comment can be removed because it is no more informative than the method name `GetSequenceNumber()`.